dist/dev/css/main-index.css

```diff
@@ -254,16 +252,15 @@ https://stackoverflow.com/a/8539107/6174302
 }
 
 /* ========================= *
- * PRINT STYLES
+ * LOADING ICON STYLES
  * ========================= */
-
-.loading {
+#loading {
```

Comment:
Any reason why? I trust you on this, but wouldn't mind knowing your reasoning

Reply:
Reasoning is about a 7 on the scale where 0 = best practices and 10 = whatever is the better term than best practices. Too lazy to summarize it, but [CSS-Tricks is dope](https://css-tricks.com/the-difference-between-id-and-class/).

Reply:
Okay, so basically they're unique so it's easier to search? Makes sense if so